features/plugin-uninstall.feature

```diff
@@ -27,12 +27,45 @@ Feature: Uninstall a WordPress plugin
 
   Scenario: Attempting to uninstall a plugin that doesn't exist
     When I try `wp plugin uninstall edit-flow`
+    And STDERR should be:
+      """
+      Warning: The 'edit-flow' plugin could not be found.
+      Error: No plugins uninstalled.
+      """
+    And the return code should be 1
+
+  Scenario: Uninstall all installed plugins
+    When I run `wp plugin uninstall --all`
     Then STDOUT should be:
       """
-      Success: Plugin already uninstalled.
+      Uninstalled and deleted 'akismet' plugin.
+      Uninstalled and deleted 'hello.php' plugin.
+      Success: Uninstalled 2 of 2 plugins.
+      """
+    And the return code should be 0
+
+    When I run the previous command again
+    Then STDOUT should be:
       """
+      Success: No plugins installed.
+      """
+
+  Scenario:  Uninstall all installed plugins when one or more activated
+    And I run `wp plugin activate --all`
+    Then STDOUT should contain:
+      """
+      Success: Activated 2 of 2 plugins.
+      """
+    And I run `wp plugin uninstall --all`
```

Comment:
As @schlessera mentioned need this to be `When I try` as it will fail without the `--deactivate` flag. Then have another successful step with the `--deactivate` flag.